Makefile

```diff
@@ -199,12 +199,16 @@ build/src/code/fmodf.o: OPTFLAGS := -g
 build/src/code/__osMemset.o: OPTFLAGS := -g
 build/src/code/__osMemmove.o: OPTFLAGS := -g
 
-# Use signed chars instead of unsigned for code_800EC960.c (needed to match AudioDebug_ScrPrt)
-build/src/code/code_800EC960.o: CFLAGS += -signed
+build/src/code/audio/%.o: OPTFLAGS := -O2
+
+# Use signed chars instead of unsigned for general.c (needed to match AudioDebug_ScrPrt)
```

Comment:
```suggestion
# Use signed chars instead of unsigned (needed to match AudioDebug_ScrPrt)
```
that can be infered from `build/src/code/audio/general.o` and since we renamed it to general.c it's not very relevant info
or
```suggestion
# Use signed chars instead of unsigned for src/code/audio/general.c (needed to match AudioDebug_ScrPrt)
```